src/nwb_datajoint/spikesorting/merged_sorting_extractor.py

```diff
@@ -0,0 +1,93 @@
+from typing import Dict, List, Union
+
+import numpy as np
+import spikeinterface as si
+
+
+class MergedSortingExtractor(si.BaseSorting):
```

Comment:
Perhaps we should submit this as a PR to spikeinterface

Reply:
Alessio is working on this, but I agree that we could do so to help him out / get it started.